gamma_bridge/settings.py

```diff
@@ -17,3 +17,5 @@
 
 
 GAMMA_API_SUFFIX = '/api/{}/gamma-profile/'.format(GAMMA_API_VERSION)
+GAMMA_FIRST_SLEEP_INTERVAL = 2
```

Comment:
Add TODO please to move this settings into the Edx settings.
@AndreyLykhoman

Reply:
Done